testsuite/domain/src/test/resources/host-configs/host-slave.xml

```diff
@@ -82,7 +82,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
IgnoredResourcesTestCase uses this config, and the main point of the test is to ensure that things explicitly listed for being ignored (the ignored-resources children below) are actually ignored. So for that test, ignore-unused-configuration="false" is definitely needed; otherwise the explicit listing stuff isn't being tested any more.